core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -70,6 +70,8 @@
 
     private static final String ENDPOINT_SUFFIX = "gateway";
 
+    private static final String INTERNAL_DISABLE_TAG_KEY = "cloudera:internal:disable-loadbalancer";
```

Comment:
Please change the variable name INTERNAL_DISABLE_TAG_KEY to something that gives a sense that is used to disable load balancer.